### PR TITLE
Areahierarchy

### PR DESCRIPTION
**Pull request by TANVIRWD**
customerUi

### CODE REVIEW

File: client-web/src/app/app-routing/app-routing.module.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ import { UsersManageComponent } from '../admin/components/users-manage/users-man
 import { RolesComponent } from '../admin/components/roles/roles.component';
 import { ServiceHierarchyComponent } from '../admin/components/service-hierarchy/service-hierarchy.component';
 import { MediaUploadComponent } from '../admin/components/media-upload/media-upload.component';
+import {CustomeruiComponent} from '../components/customerui/customerui.component';
```
Comment (sharif-i2gether) [inline client-web/src/app/app-routing/app-routing.module.ts:20]: Change customerui to profile

File: client-web/src/app/app-routing/app-routing.module.ts [hunk 2]
```diff
@@ -40,6 +41,10 @@ const routes: Routes = [
     {
       path: 'service-categories',
       component:ServiceCategoriesComponent
+    },
+    {
+      path: 'customerui',
+      component:CustomeruiComponent
```
Comment (sharif-i2gether) [inline client-web/src/app/app-routing/app-routing.module.ts:47]: Same comment

File: client-web/src/app/app.module.ts [hunk 3]
```diff
@@ -50,6 +50,8 @@ import { RoleDetailComponent } from './admin/components/roles/role-detail/role-d
 import { ServiceHierarchyAddComponent } from './admin/components/service-hierarchy-add/service-hierarchy-add.component';
 import { ServiceHierarchyEditComponent } from './admin/components/service-hierarchy-edit/service-hierarchy-edit.component';
 import { ServiceHierarchyDetailComponent } from './admin/components/service-hierarchy-detail/service-hierarchy-detail.component';
+import { CustomeruiComponent } from './components/customerui/customerui.component';
+
```
Comment (sharif-i2gether) [inline client-web/src/app/app.module.ts:54]: Same

File: client-web/src/app/components/header/header.component.html [hunk 4]
```diff
@@ -33,6 +33,18 @@
                 <button mat-button (click)="hLayer2(menuChild.id)" *ngFor="let menuChild of serviceHLayer2"> {{menuChild.title}} </button>
                 
                 <!-- <mat-icon *ngIf="serviceHLayer3?.length">expand_more</mat-icon> -->
+
+
+            <div class="showHousehold">
```
Comment (sharif-i2gether) [inline client-web/src/app/components/header/header.component.html:38]: Why the class name is showHousehold.. Household is not something static. Its dynamic. Change classname to relevant name

**Review by sharif-i2gether — CHANGES_REQUESTED** [review]

**Review by TANVIRWD — COMMENTED** [review]
done

**Pull request closed without merging**